checker/src/org/checkerframework/checker/upperbound/UpperBoundTransfer.java

```diff
@@ -81,31 +79,6 @@ public UpperBoundTransfer(CFAnalysis analysis) {
         return result;
     }
 
-    /** Makes array.length have type LTEL(array). */
```

Comment:
It's redundant with the code you added to apply the ATF; Suzanne mentioned it should be removed in an email she sent to us about viewpoint stuff last week.